client/src/components/misc/DownloadParsec.vue

```diff
@@ -0,0 +1,99 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div
+    button
+    class="download-parsec"
+  >
+    <div
+      button
+      class="download-parsec-content"
+      @click="Env.Links.openDownloadParsecLink"
+    >
+      <ms-image
+        :image="LogoParsecApp"
+        class="download-parsec-icon"
+      />
+      <ion-text class="body download-parsec-text"> {{ $msTranslate('HomePage.topbar.downloadParsec') }} </ion-text>
```

Comment:
Text is too big in french and wraps to a second line on hover  

https://github.com/user-attachments/assets/3949c813-9f0a-4175-8125-c8ac88e08152